docs-src/content/cli/Formats.md

```diff
@@ -0,0 +1,289 @@
+            "word": "just",
+            "confidence": 0.622
+          },
+          {
+            "start_time": 10664,
+            "duration": 114,
+            "word": "a",
+            "confidence": 0.744
+          },
+          {
+            "start_time": 10778,
+            "duration": 532,
+            "word": "minute.",
+            "confidence": 1
+          }
+        ],
+        "start_time": 8880,
+        "duration": 2430,
+        "confidence": 0.461,
+        "transcript": "Certainly please hold on for just a minute."
+      }
+    ]
+  }
+]
+```
+</details>
+
+## Json and Json-pretty
+
+Both `json` and `json-pretty` return the same data.
+The only difference is that `json-pretty` includes line breaks and indentation to make it more human-readable.
+Both formats return all alternatives recognized for each result, with the default JSON encoding for
+the [RecognitionAlternative](/sdk-cubic/protobuf/autogen-doc-cubic-proto/#message-recognitionalternative) 
+object. That encoding makes it straightforward to deserialize start_time and duration fields into 
+[google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) objects, but may not be as human-readable because the JSON encoding for
+Duration objects has separate fields for seconds and nanoseconds.
+
+Word-level information is not included and the results are the order cubic processed them, so results from 
+different channels are not guaranteed to be ordered according to their timestamps.
+
+### Example
+```bash
+bin/cubic-cli -s demo-cubic.cobaltspeech.com:2727 transcribe -m 1 -f json-pretty testdata/this_is_a_test-en_us-16.wav 
+```
+
+<details>
+<summary><font color="236ecc"><b>Click here to see sample output</b></font></summary>
+
+```json
+{
+  "UttID": "this_is_a_test-en_us-16.wav",
+  "Responses": [
+    {
+      "alternatives": [
+        {
+          "transcript": "This is a test.",
+          "confidence": 0.994,
+          "start_time": {
+            "nanos": 900000000
+          },
+          "duration": {
+            "seconds": 1,
+            "nanos": 50000000
+          }
+        },
+        {
+          "transcript": "Hi this is a test.",
+          "confidence": 0.006,
+          "start_time": {
+            "nanos": 5000000
+          },
+          "duration": {
+            "seconds": 1,
+            "nanos": 945000000
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+</details>
+
+## Utterance-json
+
+Utterance-json is a tab-delimited format, included for backward compatibility with asr_tool, the previous command-line test utility for cubic. 
+
+Each result is a separate line with the identifier separated from the json representation of the result 
+by a tab character:
+```
+<utteranceId>_<resultIndex> <tab> {<result json>}
+```
+
+### Example
+```bash
+bin/cubic-cli -s demo-cubic.cobaltspeech.com:2727 transcribe -m 1 -f utterance-json --list-file testdata/list.txt 
```

Comment:
It might be nice to show the contents of `testdata/list.txt` here, or provide a [link](https://github.com/cobaltspeech/cubic-cli/blob/master/testdata/list.txt) to where they can view it.